.travis.yml

```diff
@@ -17,6 +17,11 @@ addons:
     update: false
 
 script:
-- set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace Example/TABTestKit.xcworkspace -scheme TABTestKit_ExampleUITests -sdk $IPHONE_SIMULATOR_SDK -destination "OS=$OS_VERSION,name=$IPHONE_NAME" | xcpretty
-- carthage build --archive
-- pod lib lint
+  # Ensure SPM builds
+  - set -o pipefail && xcodebuild -workspace package.xcworkspace -scheme TABTestKit -sdk $IPHONE_SIMULATOR_SDK | xcpretty
```

Comment:
package.workspace is actually generated by Xcode automatically when you open a Package.swift file. It's generated in a hidden folder (.swiftpm) but if you drag it out, you can check it into the repo and it can then serve as the workspace to build the package using CI

Reply:
Hmm actually I think this can be improved by restructuring the entire repo so that the example project is in the root. I'll do that tomorrow.